themes/src/main/resources-community/theme/base/login/messages/messages_es.properties

```diff
@@ -65,12 +74,40 @@ region=Estado, Provincia, o Regi\u00F3n
 postal_code=C\u00F3digo Postal
 country=Pa\u00EDs
 emailVerified=Email verificado
-gssDelegationCredential=GSS Delegation Credential
+gssDelegationCredential=Delegaci\u00F3n de credenciales GSS
+
+profileScopeConsentText=Perfil de usuario
+emailScopeConsentText=Direcci\u00F3n de correo electr\u00F3nico
+addressScopeConsentText=Direcci\u00F3n
+phoneScopeConsentText=N\u00FAmero de telefono
+offlineAccessScopeConsentText=Acceso offline
+samlRoleListScopeConsentText=Mis roles
+rolesScopeConsentText=Roles de usuario
+
+restartLoginTooltip=Reiniciar inicio de sesi\u00F3n
 
-loginTotpStep1=Instala <a href=\"https://freeotp.github.io/\" target=\"_blank\">FreeOTP</a> o Google Authenticator en tu tel\u00E9fono m\u00F3vil. Ambas aplicaciones est\u00E1n disponibles en <a href=\"https://play.google.com\">Google Play</a> y en la App Store de Apple.
+loginTotpIntro=Usted necesita configurar un generador de contrase\u00F1as de un solo uso, para acceder a esta cuenta
```

Comment:
"Usted" not necessary. Also not necessary the colon after "uso".
"Necesita configurar un generador de contrase\u00F1as de un solo uso para acceder a esta cuenta"